docs/basic/crosshair.md

```diff
@@ -0,0 +1,35 @@
+# CrossHair
+
+```{warning}
+CrossHair is an **experimental** tool and it runs your code. So, use it only with safe functions, don't run it on the code that may wipe out your system or do bank transactions.
+```
+
+[CrossHair](https://github.com/pschanely/CrossHair) is a third-party tool for finding bugs in Python code with deal support. It is a verifier-driven fuzzer (or "[concolic testing]"), something in between deal {doc}`/basic/tests` and {doc}`/basic/verification`. It calls the given function multiple times but instead of actual values it passes special mocks, allowing it explore different execution branches.
+
+Installation:
+
+```bash
+python3 -m pip install --user crosshair-tool
+```
+
+Usage:
+
+```bash
+python3 -m crosshair check \
+    --analysis_kind=deal \
+    --report_all
```

Comment:
Hmm. Now I see. I misinterpreted what `watch` means there. I expected it to be the same as `check` but instead of checking the whole file check only the functions that you modify. This is how usually watch mode works in other tools.